src/net.h

```diff
@@ -586,6 +586,19 @@ class CNode
         }
     }
 
+    template <typename T1, typename T2, typename T3, typename T4, typename T5, typename T6, typename T7, typename T8, typename T9, typename T10, typename T11, typename T12, typename T13>
+    void PushMessage(const char* pszCommand, const T1& a1, const T2& a2, const T3& a3, const T4& a4, const T5& a5, const T6& a6, const T7& a7, const T8& a8, const T9& a9, const T10& a10, const T11& a11, const T12& a12, const T13& a13)
+    {
+        try {
+            BeginMessage(pszCommand);
+            ssSend << a1 << a2 << a3 << a4 << a5 << a6 << a7 << a8 << a9 << a10 << a11 << a12 << a13;
+            EndMessage();
+        } catch (...) {
```

Comment:
do you have something specific to catch? In such way we will lose exception message

Reply:
It is additional template for internal messages processing. Since I have added the argument to "dsee" message new template is required. "try-catch" construction is kept as in previous 12 templates. So it will be processed in the same way